clubsandwich/ui/scrollingtextview.py

```diff
@@ -0,0 +1,108 @@
+import textwrap
+
+from clubsandwich.blt.nice_terminal import terminal
+from clubsandwich.geom import Point, Size
+from clubsandwich.ui import LabelView
+from clubsandwich.ui.layout_options import LayoutOptions
+from clubsandwich.ui.view import View
+
+
+class ScrollingTextView(View):
+  """
+  :param int value: Initial value
+  :param func callback: Called with updated values
+  :param int|None min_value: Min value or ``None``
+  :param int|None max_value: Max value or ``None``
+
+  See :py:class:`View` for the rest of the init arguments.
```

Comment:
Docstring needs to be fixed to reflect the actual function signature and class. I'll fix if you don't get to it.